### DISCUSSION TITLE
Add a parameter to enable animation when mounting a component

### DISCUSSION THREAD

**Discussion by StKostyantin** [barvian/number-flow#37] [Ideas]
Example

`
<NumberFlow value={value}  mount={true} />
`

**Comment by barvian** [comment]
Hi 👋. I'm not sure this belongs in the core library, but I'll leave it open as a discussion for visibility. The current recommendation is to do it with an effect e.g.

```jsx
export default function AnimatedNumberOnMount() {
  const [value, setValue] = React.useState(0)
  React.useEffect(() => setValue(123.4), [])

  return <NumberFlow value={value} />
}
```

[Sandbox](https://codesandbox.io/p/sandbox/number-flow-react-forked-qln338?workspaceId=cbc394d8-76be-4eee-b6ad-557eaf0d663f)
